tests/storage/hotplug.go

```diff
@@ -377,6 +377,87 @@ var _ = SIGDescribe("Hotplug", func() {
 		)
 	})
 
+	Context("WFFC storage", func() {
+		var (
+			vm *kubevirtv1.VirtualMachine
+		)
+
+		BeforeEach(func() {
+			hasWffc := tests.HasBindingModeWaitForFirstConsumer()
+			if !hasWffc {
+				Skip("Skip no local wffc storage class available")
+			}
+
+			template := tests.NewRandomFedoraVMIWitGuestAgent()
```

Comment:
i'm curious, do you need the guest agent for this test?

Reply:
I guess I don't technically need the guest agent for any of the hotplug tests. But I need fedora for its hotplug support, and I have 3 available options (without making a new one):
- NewRandomFedoraVMIWitGuestAgent
- NewRandomFedoraVMIWithDmidecode
- NewRandomFedoraVMIWithVirtWhatCpuidHelper
The guest agent one seemed the least invasive. I suppose I can make a new NewRandomFedoraVMI without the guest agent.